draft-ietf-httpbis-compression-dictionary.md

```diff
@@ -99,20 +103,19 @@ and "type".
 ### match
 
 The "match" value of the Use-As-Dictionary header is a String value that
-provides the URLPattern to use for request matching
-(https://urlpattern.spec.whatwg.org/).
+provides the {{URLPattern}} to use for request matching.
```

Comment:
"[URLPattern]" is not a word, it's a citation.  You want:

```suggestion
provides the URL Pattern {{URLPattern}} to use for request matching.
```

Reply:
Thanks. I cleaned that up and changed the reference in the CORS section to point to the matching section of the doc instead of directly to URLPattern.

I'm assuming it is cleaner to not cite every reference to URL Pattern in the doc and doing it in the sections where it is directly used is the way to go but let me know if that's not the case and I can add the citations to the 2 `URL Pattern` references that I removed it from.